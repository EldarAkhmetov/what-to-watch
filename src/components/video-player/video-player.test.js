import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from './video-player.jsx';

it(`VideoPlayer correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<VideoPlayer
      isPlaying={false}
      src={`https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`}
      properties={{}}
      poster={`img/johnny-english.jpg`}
    />,
    {
      createNodeMock: (element) => {
        if (element.type === `video`) {
          return {
            src: null
          };
        }
        return null;
      }
    })
    .toJSON();

  expect(tree).toMatchSnapshot();
});
