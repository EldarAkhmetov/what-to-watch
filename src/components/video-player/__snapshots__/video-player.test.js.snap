// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer correctly renders after relaunch 1`] = `
<video
  poster="img/johnny-english.jpg"
/>
`;
